examples/shopping_ads/approve_merchant_center_link.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demonstrates how to approve a Merchant Center link request.
+
+Prerequisite: You need to have access to a Merchant Center account. You can find
+instructions to create a Merchant Center account here:
+https://support.google.com/merchants/answer/188924.
+
+To run this code example, you must use the Merchant Center UI or the Content
+API for Shopping to send a link request between your Merchant Center and Google
+Ads accounts.
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.api_core import protobuf_helpers
+
+
+def main(client, customer_id, merchant_center_account_id):
+    """Demonstrates how to approve a Merchant Center link request.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: The client customer ID string.
+        merchant_center_account_id: ID of the Merchant center whose link request
+        is to be approved.
+    """
+    merchant_center_link_service = client.get_service(
+        "MerchantCenterLinkService", version="v5"
+    )
+
+    try:
+        # Retrieve all the existing Merchant Center links.
+        response = merchant_center_link_service.list_merchant_center_links(
+            customer_id
+        )
+        print(
+            f"{len(response.merchant_center_links)} Merchant Center link(s) "
+            "found with the following details:"
+        )
+
+        merchant_center_link_status_enum = client.get_type(
+            "MerchantCenterLinkStatusEnum", version="v5"
+        )
+
+        # Iterate through the results and filter for links with pending states.
+        for merchant_center_link in response.merchant_center_links:
+            print(
+                f"Link '{merchant_center_link.resource_name}' has status "
+                f"'{merchant_center_link.status}'."
```

Comment:
Print the status as string instead of the raw enum value

Reply:
Fixed.